crates/conformance-test-utils/src/madara.rs

```diff
@@ -0,0 +1,85 @@
+use eyre::Result;
+use serde_json::Value;
+use starknet::core::types::FieldElement;
+use starknet::core::utils::get_storage_var_address;
+
+use crate::Felt;
+
+pub const FEE_TOKEN_ADDRESS_HEX: &str = "0x49d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7";
+
+/// Generates the genesis storage data for pre-funding a Starknet address on Madara.
+///
+/// This function calculates the storage key for the balance of the ERC20 Fee Token
+/// contract using the provided Starknet address. The resulting JSON object represents the initial
+/// storage of the Fee Token contract, where the account associated with the Starknet address is
+/// pre-funded with the specified `amount`.
+///
+/// # Arguments
+///
+/// * `starknet_address` - The Starknet address to be funded
+/// * `amount` - The amount of funds to allocate to the Starknet account.
+///
+/// # Returns
+///
+/// * `Result<Value>` - A JSON object representing the initial storage of the Fee Token contract
+///   where the Starknet address has been pre-funded with the specified `amount`. Returns an error
+///   if the constant `FEE_TOKEN_ADDRESS_HEX` could not be converted to a `FieldElement` or if there
+///   was an error calculating the storage variable address.
+pub fn genesis_fund_starknet_address(starknet_address: FieldElement, amount: FieldElement) -> Result<Value> {
+    // Compute the storage key for `ERC20_balances` and the Starknet address
+    let storage_var_address = get_storage_var_address("ERC20_balances", &[starknet_address])?;
```

Comment:
Entrypoint. Storage key is calculated using `ERC20_balances` as the variable name and `starknet_address` as an argument